#! /usr/bin/env python

PACKAGE = 'obstacle_collision_checker'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator ()

gen.add("delay_time", double_t, 0, "footprint_margin", 0.3, 0.0, 2.0)
gen.add("footprint_margin", double_t, 0, "footprint_margin", 0.0, 0.0, 1.0)
gen.add("max_deceleration", double_t, 0, "footprint_margin", 2.0, 0.0, 4.0)
gen.add("resample_interval", double_t, 0, "resample_interval", 0.3, 0.01, 3.0)
gen.add("search_radius", double_t, 0, "resample_interval", 5.0, 0.1, 10.0)

exit (gen.generate (PACKAGE, "obstacle_collision_checker", "ObstacleCollisionChecker"))
